### PR TITLE
Add condensed milk recipe, adjust condensed milk

### PR DESCRIPTION
**Pull request by anothersimulacrum**
#### Summary
```SUMMARY: Content "Added recipe for condensed milk, change calorie and portion count"```  

#### Purpose of change
Condensed milk has no crafting recipe currently, and it's relatively easy to make, so I added a recipe. The description was also changed to reflect that it is not always canned. Additionally, the calorie count for condensed milk is far lower than it should be, so I changed it to what it should be and increased the "charges". 

#### Describe the solution
Adds four news recipes for condensed milk: plain, and versions for all the types of jars/cans.
Condensed milk was changed to reflect it's actual calorie count, following these calculations:

1. [100g of Condensed milk contains 321kcal](https://ndb.nal.usda.gov/ndb/foods/show/01095?fgcd=&manu=&format=&count=&max=25&offset=&sort=default&order=asc&qlookup=01095&ds=&qt=&qp=&qa=&qn=&q=&ing=)
2. [One liter of condensed milk weighs 1293.39g](https://www.aqua-calc.com/page/density-table/substance/milk-coma-and-blank-canned-coma-and-blank-condensed-coma-and-blank-sweetened)
3. One 250 mL portion of condensed milk weighs 1293.39/4 = 323g
4. One gram of condensed milk contains 321/100 = 3.21kcal
5. One 250mL portion of condensed milk contains 3.21*323 &#8773; 1038kcal

Also increase the charges of condensed milk: There are ten portions of condensed milk per 250mL so that calorie count remains somewhat similar and changes do not need to be made to recipes.

 ##### TODO:

- [x] Fix formatting
- [x] Check out condensed milk calorie values and increase charges

#### Describe alternatives you've considered
I found some other recipes using salt/butter, but there were also many recipes that just used sugar and milk, so I went with this one.

#### Additional context
[Video](https://www.youtube.com/watch?v=PgEUH_zTwB8)
[Recipe 1](https://www.biggerbolderbaking.com/how-to-make-condensed-milk/)
[Recipe 2](https://cookingshooking.com/condensedmilk/)

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
It's been reverted.

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -8317,5 +8317,96 @@
...
+    "time": 120000,
+    "result_mult": 12,
+    "book_learn": [ [ "cookbook_italian", 2 ] , [ "family_cookbook", 2 ] , [ "mag_cooking", 2 ] ],
+    "batch_time_factors": [ 80, 4 ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 200, "LIST" ] ], [ [ "pot_canning", -1 ] ] ],
+    "components": [
+      [ [ "jar_3l_glass", 1 ] ],
+      [ [ "water", 22 ], [ "water_clean", 22 ] ],
+      [ [ "milk", 12 ] ],
```
Comment (Reclusive-reptile) [inline data/json/recipes/recipe_food.json]: Needs to take twice the volume of milk boiled down to 3L, so 24 milk.

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -8317,5 +8317,96 @@
...
+    "difficulty": 3,
+    "time": 45000,
+    "result_mult": 2,
+    "book_learn": [ [ "cookbook_italian", 2 ] , [ "family_cookbook", 2 ] , [ "mag_cooking", 2 ] ],
+    "batch_time_factors": [ 80, 4 ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 200, "LIST" ] ], [ [ "pot_canning", -1 ] ] ],
+    "components": [
+      [ [ "jar_glass", 1 ] ],
+      [ ["milk", 2] ],
```
Comment (Reclusive-reptile) [inline data/json/recipes/recipe_food.json]: 4 milk.

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -8317,5 +8317,96 @@
...
+      { "id": "SAW_M", "level": 1 },
+      { "id": "HAMMER", "level": 1 },
+      { "id": "CUT", "level": 1 },
+      { "id": "COOK", "level": 3 }
+    ],
+    "tools": [ [ [ "surface_heat", 10, "LIST" ] ], [ [ "can_sealer", -1 ] ] ],
+    "components": [
+      [ [ "canister_empty", 1 ], [ "can_food_unsealed", 1 ] ],
+      [ [ "scrap", 1 ] ],
+      [ [ "milk", 1 ] ],
```
Comment (Reclusive-reptile) [inline data/json/recipes/recipe_food.json]: 2 milk.

**Comment by KorGgenT** [comment]
Take a look at the calorie count in condensed milk IRL while you're at it. you'll find that it should have about 10x the amount; with that, it could have 10x the portions (being smaller volume) and still be usable in recipes that use regular milk without being strange.

File: data/json/recipes/recipe_food.json [hunk 4]
```diff
@@ -8317,5 +8317,96 @@
...
+  },
+  {
+    "type": "recipe",
+    "result": "con_milk",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": 40000,
+    "book_learn": [ [ "cookbook_italian", 2 ] , [ "family_cookbook", 2 ] , [ "mag_cooking", 2 ] ],
```
Comment (kevingranade) [inline data/json/recipes/recipe_food.json]:
```suggestion
    "book_learn": [ [ "cookbook_italian", 2 ], [ "family_cookbook", 2 ], [ "mag_cooking", 2 ] ],
```

File: data/json/recipes/recipe_food.json [hunk 5]
```diff
@@ -8317,5 +8317,96 @@
...
+    "batch_time_factors": [80, 4],
```
Comment (kevingranade) [inline data/json/recipes/recipe_food.json]:
```suggestion
    "batch_time_factors": [ 80, 4 ],
```

File: data/json/recipes/recipe_food.json [hunk 6]
```diff
@@ -8317,5 +8317,96 @@
...
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 120, "LIST" ] ] ],
+    "components": [
+      [ ["milk", 2] ],
```
Comment (kevingranade) [inline data/json/recipes/recipe_food.json]:
```suggestion
      [ [ "milk", 2 ] ],
```

File: data/json/recipes/recipe_food.json [hunk 7]
```diff
@@ -8317,5 +8317,96 @@
...
+      [ [ "sugar", 100 ], [ "syrup", 1 ], [ "beet_syrup", 1 ], [ "molasses", 1 ], [ "honey_bottled", 1 ], [ "honey_glassed", 1 ] ]
```
Comment (kevingranade) [inline data/json/recipes/recipe_food.json]:
```suggestion
      [
        [ "sugar", 100 ],
        [ "syrup", 1 ],
        [ "beet_syrup", 1 ],
        [ "molasses", 1 ],
        [ "honey_bottled", 1 ],
        [ "honey_glassed", 1 ]
      ]
```

**Comment by anothersimulacrum** [comment]
I've linted my changes and made those suggested by KorGgenT, so I think this should be ready.

**Comment by Ker-Nes** [comment]
The portions in condensed milk currently means you can multiply your milk, going from milk to condensed (at 2 milk to 10 condensed) and back to milk (at 1 condensed to 2 milk), to get however much of it you wish, only at the expense of some sugar.

While I suppose it could be a criticism of the nature of modern industrialized milk products, I don't quite think that's what was intended...

**Comment by Muffindrake** [comment]
The recipes all use the same amount of sugar/sugar substitute, despite varying amounts of milk.

**Pull request merged**